openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/admin/EditDialog.vue

```diff
@@ -73,12 +73,7 @@
               </v-row>
             </div>
             <v-row no-gutters>
-              <v-textarea
-                v-model="json_content"
-                rows="15"
-                :readonly="readonly"
-                data-test="editTextInput"
-              />
+              <pre id="editor"></pre>
```

Comment:
Using id is going to cause us problems eventually if we do the split screen.  Is there any way to not use id, or give it a unique id?

Reply:
Yeah I think you can select the element yourself and give it to the ace constructor. I'll see if I can make that work and update EditScreenDialog and ScriptRunner as well.